src/util/text.ts

```diff
@@ -234,10 +234,15 @@ export function isOctal(c: i32): bool {
   return c >= CharCode._0 && c <= CharCode._7;
 }
 
+/** Tests if the specified character code is a valid hexadecimal symbol [a-f]. */
+export function isHexPart(c: i32): bool {
```

Comment:
Perhaps `isHexAlpha` to hint at a-f/A-F?

Reply:
I think better rename this to `isHex` while original `isHex` rename to `isHexDecimal`

Reply:
The precise long form term for the current `isHex` would probably be `isHexadecimal`. Yet I think that `isHex` has been chosen here, and isn't uncommon as a shorter form, using it for a-f/A-F could again be confusing, indicating something like `isHexadecimalAlpha` heh

Reply:
`isHexadecimal` is `0-9` + `a-f` while `isHex` only check `a-f` without `0-9`

Reply:
We already have `isAlpha` and `isAlphaOrDecimal` and `isAlpha` if a full alphabet (`a-z`)

Reply:
My point is that you have almost naturally chosen `isHex` as a short form of `isHexadecimal`, and when I read `isHex` I would likewise not make the connection that it only covers a-f/A-F (which is what you propose), as I would similarly assume that it's just short for hexadecimal. Now I'm wondering how we could make this clearer by taking one step backwards... hmm. Perhaps `isBase10`, `isBase16` etc., then with `isAtoF` as the partial?

Reply:
The short of `isHex` instead of `isHexadecimal` makes sense to me because it's not full hex + decimal (hexadecimal). It's just part of `a-f`. So we should this call something different from `isHexadecimal`

Reply:
> Perhaps isBase10, isBase16 etc., then with isAtoF as the partial?

I don't think it's that important. These are not public methods at all. I'm inclined to leave it as it is now. The most important thing is that we ourselves are not confused about it =)

Reply:
Renamed `isHex` to `isHexBase`. Is it better?

Reply:
I guess I can live with anything indicating that this function does something a bit special, so `isHexBase` will do.